apps/website/src/pages/docs/avatar.mdx

```diff
@@ -13,74 +13,56 @@ import * as AllExamples from 'examples';
   <AllExamples.AvatarInitialsExample client:load />
 </LiveExample>
 
-### Icon
+The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials.
 
-An icon may be display instead of initials, typically used by organizations instead of a person.
+## Usage
 
-<LiveExample src='Avatar.icon.tsx'>
-  <AllExamples.AvatarIconExample client:load />
-</LiveExample>
+Different properties can be passed to the `<Avatar>` component to customize it according to the user's needs.
 
-### Picture
+- `abbreviation` prop allows users to provide a short abbreviation or initials for the avatar.
+- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
 
-If a user uploads a photo we display their photo instead of their initials.
+### Sizes
+
+Avatars can be displayed in various sizes including `small`, `medium`, `large`, and `x-large`.
 
-<LiveExample src='Avatar.picture.tsx'>
-  <AllExamples.AvatarPictureExample client:load />
+<LiveExample src='Avatar.sizes.tsx'>
+  <AllExamples.AvatarSizesExample client:load />
 </LiveExample>
 
 ### Status indicators
 
-You may want to indicate the online status of a user. Status indicators display in the bottom right of the avatar. We suggest not using statuses on the small size avatar. The available statuses are:
+Status indicators are indicators that display in the bottom right of the avatar. Statuses can be set using the `status` prop. The available statuses are:
 
 - `online`
 - `away`
 - `busy`
 - `offline`
 
+We suggest not using statuses on the small size avatar.
+
 <LiveExample src='Avatar.statuses.tsx'>
   <AllExamples.AvatarStatusesExample client:load />
 </LiveExample>
 
-## Avatar group
+## Variants
 
-Avatars can be displayed in a list to represent a group of people. This visual treatment reinforces the fact that these people work together and/or share similar permissions to access/view/edit a file or a project. For example, you might display the list of people who previously edited an item, are all collaborating on a specific activity, etc. To save space, consider the "Stacked" option below.
+An avatar can be customized to show initials, icons, and pictures.
 
-<LiveExample src='Avatar.group.tsx'>
-  <AllExamples.AvatarGroupExample client:load />
+<LiveExample src='Avatar.main.tsx'>
+  <AllExamples.AvatarMainExample client:load />
 </LiveExample>
 
+When no profile picture is uploaded, an avatar with the user's initials is shown. It can display one or two letters, or the first letter of their email if no name is provided. You may use an icon instead. Once a photo is uploaded, it replaces the initials/icon.
+
+### Avatar Group
+
+Avatars can be displayed in a list to represent a group of people using `<AvatarGroup>` component.
+
 ### Stacked
 
 To save on space, the avatars can overlay each other. Note that when using the stacked group that you can not display [status indicators](#status-indicators).
```

Comment:
We should mention the `stacked` prop here

Reply:
suggested rewording in https://github.com/iTwin/iTwinUI/pull/1667#discussion_r1394905397